### PR TITLE
NC: prepare for 2016 Extra Session 2 (starts 3/23)

### PR DESCRIPTION
**Pull request by Skekacs93**

### CODE REVIEW

File: openstates/nc/__init__.py [hunk 1]
```diff
@@ -83,8 +83,13 @@
                  'display_name': '2015-2016 Session',
                  '_scraped_name': '2015-2016 Session',
                 },
+        # '2016 Extra Session 2': {'start_date': datetime.date(2016,3,23), 'type': 'special',
```
Comment (loandy) [inline openstates/nc/__init__.py]: Since we're ignoring the session, no need to keep this in.

**Comment by Skekacs93** [comment]
Ok, I was ignoring it for now because the scraper won't run without it, but had the session info in there so we can easily uncomment it once it starts?

**Comment by loandy** [comment]
I debated that, but for now, I'd like to address things as they come and keep the codebase cleaner.  Also, it looks like the session key format used in the past is something like `2016E2`.  Not sure why all of those commented terms exist, but I'm inclined to remove them eventually.

**Comment by Skekacs93** [comment]
Alright sounds good.

**Comment by loandy** [comment]
Thanks!

**Pull request merged**